ulid/__init__.py

```diff
@@ -7,7 +7,7 @@
 from collections.abc import Callable
 from datetime import datetime
 from datetime import timezone
-from typing import Any
+from typing import Any, Generic, Self, TypeVar
```

Comment:
The linter will complain about this. You can run
```bash
$ hatch run lint:fmt
```
to automatically fix this.

To install `hatch` I would recommend something like [pipx](https://pypa.github.io/pipx/)

Reply:
Thanks, done!